web/static/js/utils.js

```diff
@@ -0,0 +1,18 @@
+var $ = require("jquery");
```

Comment:
All of the jquery in this file is bad. We shouldn't expect the data going into `setupStreams` to be jquery object. That function should probably be passed in functions for getting the value of an element and triggering a changing on that element.